src/assets/CollectiblesController.ts

```diff
@@ -815,6 +834,8 @@ export class CollectiblesController extends BaseController<
    * @param options.getERC721OwnerOf - Get the owner of a ERC-721 collectible.
    * @param options.getERC1155BalanceOf - Gets balance of a ERC-1155 collectible.
    * @param options.getERC1155TokenURI - Gets the URI of the ERC1155 token at the given address, with the given ID.
+   * @param options.onCollectibleAdded - Callback that is called when a collectible is added. Currently used pass data
```

Comment:
What if we updated the name `onCollectibleAdded` to something like `getDataOnCollectibleAdded` or `onCollectibleAddedCallback`?

Reply:
Open to this.

Reply:
erm...please ignore my last comment. After looking through the code more, I realize that the prepending of `on...` is a common pattern for our controller listener callbacks. So `onCollectibleAddedCallback` might be redundant, and starting the name with `get` might throw people off. 

I'm pondering ways to indicate that this method returns a value while also being a callback. Maybe `onCollectibleAddedGetData` or `onCollectibleAddedReturnData`?

This is completely optional.